pkg/operator/ceph/cluster/mon/service.go

```diff
@@ -67,6 +67,15 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 		return "", nil
 	}
 
```

Comment:
Since they will only be able to enable this setting if the latest ceph version with the fix is available, how about we add a check here for ceph version 17.2.6? Otherwise, we just fail the reconcile.

Reply:
do we still need this?

Reply:
add this check in the beginning when the cluster is loading (`configureLocalCephCluster`)